addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -13,12 +13,17 @@ options.registry.WebsiteEvent.include({
      */
     allowRoomCreation(previewMode, widgetValue, params) {
         this._rpc({
-            model: this.modelName,
+            model: "event.event",
             method: 'write',
-            args: [[this.eventId], {
-                meeting_room_allow_creation: widgetValue
-            }],
-        }).then(() => this.trigger_up('request_save', {reload: true, optionSelector: this.data.selector}));
+            args: [
+                [this.eventId], {
+                    meeting_room_allow_creation: widgetValue && widgetValue !== "",
+                }
+            ],
+        }).then(() => this.trigger_up('request_save', {
+            reload: true,
```

Comment:
Actually I'm noticing it's not actually updated with just reload. `reloadWebClient: true` might make sense